definitions/npm/algoliasearch_v3.x.x/flow_v0.56.x-/algoliasearch_v3.x.x.js

```diff
@@ -169,33 +169,39 @@ declare module 'algoliasearch' {
   declare type ClientOptions = $algoliasearch$ClientOptions;
   declare type Index = $algoliasearch$Index;
   declare type Settings = $algoliasearch$Settings;
-  declare module.exports: (
-    applicationID: string,
-    apiKey: string,
-    options?: ClientOptions
-  ) => Client;
+  declare module.exports: {
+    (
+      applicationID: string,
+      apiKey: string,
+      options?: ClientOptions
+    ): Client;
+  }
```

Comment:
I'd be curious to see, is there any documentation that guided your fix here? I think this would be useful to know for other libdefs if they're required asynchronously.

Reply:
Not really, it was a bit of trial and error to get it working. I saw the syntax used somewhere, but I can't remember where.